crates/core/src/fnc/string.rs

```diff
@@ -21,6 +21,25 @@ fn limit(name: &str, n: usize) -> Result<()> {
 	Ok(())
 }
 
+pub fn capitalize((string,): (String,)) -> Result<Value> {
+	if string.is_empty() {
+		return Ok(string.into());
+	}
+	// Capitalize first character of each word (title case)
+	let words: Vec<&str> = string.split_whitespace().collect();
```

Comment:
question: wouldn't that pose problem with strings that contains multiple whitespaces? Also maybe ` ` is not necessarily the only char that is considered a whitespace?

Good thing is that you can add tests for this. ;)

Reply:
Fixed in https://github.com/surrealdb/surrealdb/pull/6452/commits/61dadce5740122a9b9e264e852881930e9056f69